specification/frontdoor/resource-manager/Microsoft.Network/stable/2025-10-01/networkexperiment.json

```diff
@@ -0,0 +1,1443 @@
+          },
+          {
+            "in": "body",
+            "name": "parameters",
+            "description": "An Network Experiment Profile",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Profile"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/Profile"
+            }
+          },
+          "201": {
+            "description": "Created. The request has been fulfilled and a new protection policy has been created.",
+            "schema": {
+              "$ref": "#/definitions/Profile"
+            }
+          },
+          "202": {
+            "description": "Accepted. The request has been accepted for processing and the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/Profile"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Creates an NetworkExperiment Profile in a Resource Group": {
+            "$ref": "./examples/NetworkExperimentCreateProfile.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "NetworkExperimentProfiles"
+        ],
+        "description": "Updates an NetworkExperimentProfiles",
+        "summary": "Updates an NetworkExperimentProfiles by NetworkExperimentProfile name",
+        "operationId": "NetworkExperimentProfiles_Update",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ProfileNameParameter"
+          },
+          {
+            "in": "body",
+            "name": "parameters",
+            "description": "The Profile Update Model",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ProfileUpdateModel"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. successful operation",
+            "schema": {
+              "$ref": "#/definitions/Profile"
+            }
+          },
+          "202": {
+            "description": "Accepted. The request has been accepted for processing and the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/Profile"
```

Comment:
>$ref [](http://example.com/codeflow?start=15&length=4)

202 for PATCH should not return body. See [https://github.com/Azure/azure-openapi-validator/blob/main/docs/patch-response-codes.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/patch-response-codes.md)

Reply:
For new RTs only.